server/backend/backend.go

```diff
@@ -135,7 +135,7 @@ func (b *Backend) RefreshUser(userID string, refresh proton.RefreshFlag) error {
 			return err
 		}
 
-		acc.updateIDs = append(acc.updateIDs, updateID)
+		acc.updateIDs = []ID{updateID}
```

Comment:
Why do the previous events need to be dropped.

Reply:
this is how api behaves if refreshFlag == All it means we don't have the history. So all the events are dropped and you should resync and then get latest event ID.